Instructor-Guide--Assignments--Marking--Grading-View.md

```diff
@@ -43,9 +44,50 @@ The grading bar, located at the top of the grading view page, provides some usef
 
 ![Grading Bar](images/grade-view-grading-bar.png)
 
-The left half of the bar allows you to quickly switch between assignments by clicking on the "Previous" and "Next" buttons. It also displays the group number of the group who's submission you're currently grading. Finally, on the right, it shows you how many submissions have been set to the "Complete" status.
+The left half of the bar allows you to quickly switch between submissions by clicking on the "Previous" and "Next" buttons. Filters for these submissions can be specified by graders by clicking the "Filter Submissions" button (see "[Filter Submissions](#filter-submissions)"). A grader can also view a random incomplete submission by clicking the "Random Incomplete Submission" button. Further, the grading bar displays the group number of the group who's submission you're currently grading. Finally, on the right, it shows you how many submissions have been set to the "Complete" status.
+
 The right half of the bar shows you the group's current mark for the submission (this will update as you assign and delete marks from the "marks" tab). It also allows you to set the marking for the submission to complete by clicking on the "Set to Complete" button. Once this has been done you may release the marks to the students by clicking on the "Release Marks" button (see "[Releasing Grades](Instructor-Guide--Assignments--Releasing-Grades.md)"). Finally, if you wish to switch to full screen mode, click on the "Fullscreen" button at the end of the bar.
 
+## Filter Submissions
+
+MarkUs allows you to filter and/or sort submissions based on specific criteria, such as marking state, tags, and other relevant attributes.
+To do this, click on the "Filter Submissions" button on the [Grading Bar](#grading-bar):
+
+![Filter Submissions button](images/filter-submissions-button.png)
+
+This will open up a Filter Submissions modal:
+
+![Filter Submissions](images/filter-submissions-modal.png)
+
+From here you may:
+
+- Order submissions (in ascending/descending order) by one of the following:
+    - Group Name (default).
+    - Submission Date.
+    - Total Mark.
+- Filter submissions by one or more of the following:
+    - Marking State (see [Marking State](Instructor-Guide--Assignments--Marking--Set-Up.md#marking-state)) : Filter all submissions with the specified marking state. Select one of the marking states from the dropdown:
```

Comment:
Do not include a space before the colon. Instead, ensure that all items are of the form `Marking State: ...`. Write all other parenthetical information ("see ...") in the description of the filter, not before the colon.

Also bold each filter name, that will help them stand out a bit more.